lib/errors.js

```diff
@@ -267,6 +267,35 @@ module.exports = function(config) {
 	});
 };`,
 
+		invalidFileExportReporterUsage: () => `error 21:
```

Comment:
```suggestion
		invalidFileExportReporterUsage: () => `Error 21:
```

Reply:
I did not take over this suggestion to be consistent to the existing error messages

Reply:
Fine, but could we have a central resources file (like i18n) where all error messages are stored and can be easily modified? Then we could easily fix such things without having to sacrifice consistency.